### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Cisco-the-wayword**
In this branch, I added:
- 1 Html file as requested.
- 1 Css file linked to the html file.
- A test file expected to be ignored by git.
- A more explanatory ReadME file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
+# Hello-Microverse-Project
+ Hello Microverse
+
+ Built with:
+HTML,
+CSS.
+
+ Technologies used:
+Github,
+Vscode.
```
Comment (omarsalem7) [inline README.md]: - [ ] Kindly use [this template](https://github.com/microverseinc/readme-template) and adjust it for your project.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# This file should be ignored by git
```
Comment (omarsalem7) [inline test.md:1]: - [ ] Also consider adding `.gitignore` file and add this file inside it beside `node_modules`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Cisco-the-wayword 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Cisco-the-wayword) [reply README.md]: Thank you.
I have gone through the corrections.
How does it look now?

File: test.md (hunk 2 above)
Comment (victorokoroji) [reply test.md:1]: - [ ] Kindly add `.gitignore` and not` random.gitignore` just as the previous reviewer suggested 👍🏼 .

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Cisco-the-wayword , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Cisco-the-wayword, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

- [ ] kindly ensure the test.md file is not visible in your repo to meet the project requirement.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**